src/hooks/useIsContributor.ts

```diff
@@ -0,0 +1,19 @@
+import useJoinedColonies from './useJoinedColonies.ts';
+
+const useIsContributor = ({
+  walletAddress = '',
+  colonyAddress = '',
+}: {
+  walletAddress: string | undefined;
+  colonyAddress: string | undefined;
+}) => {
+  const { joinedColonies, loading } = useJoinedColonies(walletAddress);
+
+  const isContributor = joinedColonies
```

Comment:
maybe we could use [array.some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)

Reply:
Great shout @bassgeta ! Pushed!